### ISSUE TITLE
Program Committee Suggestions/Confirmation

### ISSUE THREAD

**Issue by alvations** [nlposs/NLP-OSS#1]
Let's use this issue to track 
 - suggestions on new program committee members
 - how many confirmed program committee members we have.

**Comment by alvations** [comment]
Let's streamline the confirmation of PCs and invitation to people we've missed. 

 - Anyone on the shortlist of the proposal liking (thumbs-up) this issue automatically confirms his participation as program committee. 
   - Let's also add the affiliations once they're confirmed. 

 - Anyone who wants to participate but not in the shortlist, comment in this issue. Then we'll validate and add you to the list if any of the current shortlisted PC  give you a thumbs-up. (let's count that as a 'recommendation' from the shortlisted PCs)

 - We'll add anyone that the shortlisted PCs suggested/thumbs-uped to the PC shortlist.

**Comment by alvations** [comment]
Sorry for the change for automatic confirmation from ~~giving the issue comment thumbs up~~ to **starring the repository**.  

It's because Github collapses the usernames in the list of users that gave the 👍 and we couldn't check who has automatically confirmed by giving the 👍. 

Please ⭐️ the repository instead if you're on the shortlist and wants to confirm your participation as program committee. 

Also, ⭐️ the repo if you like the workshop idea! And comment in this issue if you would like to participate as program committee but we missed you in the shortlist.

**Comment by amueller** [comment]
Hey there.
This sounds like an awesome workshop. I'm not really an NLP researcher, so I'm unlikely to be at any of the conferences this is colocated with. @jnothman or @vene might. I'm happy to help review papers, but I'm probably too busy to do anything beyond that.

**Comment by dimazest** [comment]
That's great. @amueller you might want to join the program committee to review some papers.

Also, if you have suggestion for the invited speakers, let us know.

**Comment by alvations** [comment]
Awesome! @amueller we'll count you in as our program committee PC! 😊

**Comment by jnothman** [comment]
Thanks for the ping, @amueller. I'd be very keen on joining in this conversation (and am interested in any related conversation in ML FOSS). I hope to attend ACL'18 if this happens to land up there.

**Comment by alvations** [comment]
Thanks @jnothman! Can we count that as a yes to the PC invitation? 

Would you be interested in giving a talk on scikit-learn and ML/NLP too? @dimazest would follow you up on this 😉

**Comment by jnothman** [comment]
@bethard may be interested.

**Comment by ninjin** [comment]
Sure, count me in.  We do need to discuss tooling far more than we currently do.

**Comment by alvations** [comment]
Awesome @ninjin! I've confirmed you on the PC list =)

**Comment by evanmiltenburg** [comment]
So just to be sure: do PC members only review papers, or are there other things you'd like members to do?

**Comment by bethard** [comment]
I'm probably interested. What's the timeframe? I'm on family leave this semester, but should be mostly back to normal in spring.

**Comment by alvations** [comment]
@evanmiltenburg yes, as a PC member, you will help to review the papers and also spread the word about the workshop to your colleagues and encourage people to submit relevant papers to the workshop =)

@bethard Spring should be a good time when we'll push for call for papers and reviewing will start around then too. 

Our proposal would be submitted to one of the four conferences and the workshop would be co-located with either: 

|Conference | Location | Paper Review Period | Conference Dates |
|:----|:----|:----|
| ACL  | Melbourne, Australia | 1-25 April | 15-20 July |
| COLING  | Sante Fe, New Mexico, USA | 1-25 May | 20-25 August | 
| NAACL HLT | New Orleans, Louisiana, USA | 1-25 March |  1-6 June | 
| EMNLP | TBA | TBA | TBA |

The reviewing period is usually done 3 months before the conference and due 2 months before the conference.

**Comment by mefoster** [comment]
Hello all,

I have a background in both NLP (dialogue systems, NLG) and OSS (Fedora packaging mostly). I'm not as active in either community as I used to be, but I think this workshop is a great idea and I'd be glad to join the programme committee.

Mary Ellen Foster

**Comment by dimazest** [comment]
@bethard

**Comment by dimazest** [comment]
oops, wrong button was pressed

**Comment by proycon** [comment]
Thanks for the invitation. I'd gladly join the programme committee. I'm likely not in a position to attend the conference itself unfortunately.

**Comment by alvations** [comment]
@proycon Awesome! Thank you, Maarten!

**Comment by dimazest** [comment]
@proycon I'll add you to the PC list

**Comment by proycon** [comment]
Thanks!
